filebeat/config/config.go

```diff
@@ -15,7 +15,8 @@ import (
 // Defaults for config variables which are not set
 const (
 	DefaultRegistryFile                      = ".filebeat"
-	DefaultIgnoreOlderDuration time.Duration = 24 * time.Hour
+	DefaultIgnoreOlderDuration time.Duration = 72 * time.Hour
```

Comment:
@urso I suggest to change the default to inifinite (which would be 0). WDYT? Including also @guyboertje (see https://github.com/logstash-plugins/logstash-input-file/pull/87/files#r49697378)